hail/src/main/scala/is/hail/services/DeployConfig.scala

```diff
@@ -54,6 +59,7 @@ class DeployConfig(
   val location: String,
   val defaultNamespace: String,
   val serviceNamespace: Map[String, String]) {
+  import DeployConfig._
```

Comment:
Can you remind me why you need this line again? I thought you already got object methods for free.

Reply:
Sadly, one does not. If you want the methods from the object in the class without prefixing `DeployConfig.`, you have to write this.